src/ferc_xbrl_extractor/datapackage.py

```diff
@@ -405,3 +382,46 @@ def construct_dataframe(self, instance: Instance) -> pd.DataFrame:
 
         # Create dataframe and drop empty rows
         return pd.DataFrame(df).dropna(how="all")
+
+
+class Datapackage(BaseModel):
+    """A generic Data Package, as per Frictionless Data specs.
+
+    See https://specs.frictionlessdata.io/data-package.
+    """
+
+    profile: str = "tabular-data-package"
+    name: str = "ferc1-extracted-xbrl"
+    title: str = "Ferc1 data extracted from XBRL filings"
```

Comment:
Is this actually specific to the FERC 1? Or will the name and title need to be set dynamically based on which form is being extracted?

Reply:
Good catch, it was just using FERC 1 when it should be dynamically set. Fixed now

Reply:
Are these just default values now that are dynamically overridden?  Should there be default values at all if they're dynamically generated?

Reply:
Yeah the default isn't doing anything now. I've removed it